### PR TITLE
Refactor/my organization

### PR DESCRIPTION
**Pull request by wwills2**

### CODE REVIEW

File: src/renderer/components/blocks/layout/LeftNav.tsx [hunk 1]
```diff
@@ -140,10 +146,9 @@ const LeftNav = () => {
                 </div>
               </Card>
               <Sidebar.Item
-                disabled={orgCreationPending || organizationsListLoading}
                 style={{ cursor: 'pointer' }}
                 active={isActive(ROUTES.MY_ORGANIZATION)}
-                onClick={handleClickMyOrganization}
+                onClick={orgCreationPending || organizationsListLoading ? () => {} : handleClickMyOrganization}
```
Comment (MichaelTaylor3D) [inline src/renderer/components/blocks/layout/LeftNav.tsx:151]: can you make this a `noop`?

File: src/renderer/components/blocks/tabs/MyCommittedUnitsTab.tsx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  orgUid: string;
+  search: string;
+  setIsLoading: (isLoading: boolean) => void;
+}
+
+const MyCommittedUnitsTab: React.FC<PageTabProps> = ({ orgUid, search, setIsLoading }: PageTabProps) => {
+  const [currentPage, setCurrentPage] = useQueryParamState('page', '1');
+  const [order, setOrder] = useQueryParamState('order', undefined);
+  const handleSetOrder = useColumnOrderHandler(order, setOrder);
+  //todo: rename this fragment to reflect units and replace the projects modal with the units modal !!!!
```
Comment (MichaelTaylor3D) [inline src/renderer/components/blocks/tabs/MyCommittedUnitsTab.tsx:18]: did you do this TODO?

Comment (wwills2) [reply src/renderer/components/blocks/tabs/MyCommittedUnitsTab.tsx:18]: yes. I need to pull in the units modal from refactor base, but i wanted to commit prior to doing so. I dont like merging with uncommitted changes.

This todo will not exist when merged

Comment (wwills2) [reply src/renderer/components/blocks/tabs/MyCommittedUnitsTab.tsx:18]: change of plans, there will be todo's becuase the units modal still needs to be added

**Pull request merged**